open-craft/opencraft
# Use new Filebeat variables for TLS.

Testing Instructions:

1. Check this branch out on stage.
2. Restart stage.
3. Spawn a new appserver.
4. Check that the appserver spawned successfully.
5. SSH in and check that Filebeat and any other service seems to be running properly.
6. Check Kibana and filter things down to just this server and see that logs are being received.
